CMake 4.0.0 vcpkg latest 编译osg错误gperf异常

CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:127 (message):
    Command failed: D:/jungle/develop/Microsoft/VisualStudio/2022/Community/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe -v
    Working Directory: D:/jungle/develop/vcpkg/buildtrees/gperf/x64-windows-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      D:\jungle\develop\vcpkg\buildtrees\gperf\config-x64-windows-dbg-CMakeCache.txt.log
      D:\jungle\develop\vcpkg\buildtrees\gperf\config-x64-windows-rel-CMakeCache.txt.log
      D:\jungle\develop\vcpkg\buildtrees\gperf\config-x64-windows-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:269 (vcpkg_execute_required_process)
  ports/gperf/portfile.cmake:19 (vcpkg_cmake_configure)
  scripts/ports.cmake:196 (include)


error: building gperf:x64-windows failed with: BUILD_FAILED
See https://learn.microsoft.com/vcpkg/troubleshoot/build-failures?WT.mc_id=vcpkg_inproduct_cli for more information.
Elapsed time to handle gperf:x64-windows: 481 ms
Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
  https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+gperf
You can submit a new issue at:
  https://github.com/microsoft/vcpkg/issues/new?title=[gperf]+Build+error+on+x64-windows&body=Copy%20issue%20body%20from%20D%3A%2Fjungle%2Fdevelop%2Fvcpkg%2Finstalled%2Fvcpkg%2Fissue_body.md

--- a/ports/gperf/CMakeLists.txt
+++ b/ports/gperf/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.5)
 project(gperf LANGUAGES C CXX)
 
 include(CheckCXXSourceCompiles)

--- a/ports/gperf/vcpkg.json
+++ b/ports/gperf/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "gperf",
   "version": "3.1",
-  "port-version": 6,
+  "port-version": 7,
   "description": "GNU perfect hash function generator",
   "homepage": "https://www.gnu.org/software/gperf/",
   "dependencies": [

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3266,7 +3266,7 @@
     },
     "gperf": {
       "baseline": "3.1",
-      "port-version": 6
+      "port-version": 7
     },
     "gperftools": {
       "baseline": "2.16",

--- a/versions/g-/gperf.json
+++ b/versions/g-/gperf.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "ce0c8faaa002c19f8d96a64d4929439cab4570f2",
+      "version": "3.1",
+      "port-version": 7
+    },
+    {
       "git-tree": "590e19515bff5b0abc6d9f73ba20a19a71555a32",
       "version": "3.1",
       "port-version": 6

